definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-deemed-and-dispensed-nonprod.json

```diff
@@ -222,5 +222,40 @@
     "CaseStateID": "AwaitingDNApplication",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "ServicePayment",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "ServicePayment",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "ServicePayment",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "ServicePayment",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "ServicePayment",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
   }
```

Comment:
What about the authorisations for the "AwaitingServicePayment" state?

Reply:
That is out of scope as far as i believe

Reply:
@jacmurphy  If you use `AwaitingServicePayment` and this state wouldn't have proper permissions it would be implicitly in scope of your story.

@jvictorcf 
this state already exists and has permissions (CRU for beta and la, R for others):
```  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-courtadmin_beta",
    "CRUD": "CRU"
  },
  {
    "LiveFrom": "29/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-courtadmin-la",
    "CRUD": "CRU"
  },
  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-superuser",
    "CRUD": "R"
  },
  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-courtadmin",
    "CRUD": "R"
  },
  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-solicitor",
    "CRUD": "R"
  },
  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "citizen",
    "CRUD": "R"
  },
  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-systemupdate",
    "CRUD": "R"
  },
  {
    "LiveFrom": "23/07/2020",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "AwaitingServicePayment",
    "UserRole": "caseworker-divorce-bulkscan",
    "CRUD": "R"
  },
```